content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Mengoperasikan klaster etcd untuk Kubernetes
+content_type: task
+---
+
+<!-- overview -->
+
+{{< glossary_definition term_id="etcd" length="all" prepend="etcd adalah ">}}
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Prerequisites
+
+* Jalankan etcd sebagai klaster dari anggota ganjil.
+
+* etcd adalah sistem terdistribusi berbasis _leader_. Pastikan _leader_ secara berkala mengirimkan _heartbeat_ dengan tepat waktu ke semua pengikutnya untuk menjaga kestabilan klaster.
+
+* Pastikan tidak terjadi kekurangan sumber daya.
+
+  Kinerja dan stabilitas dari klaster sensitif terhadap jaringan dan _IO disk_. Kekurangan sumber daya apa pun dapat menyebabkan _timeout_ dari _heartbeat_, yang menyebabkan ketidakstabilan klaster. Etcd yang tidak stabil mengindikasikan bahwa tidak ada _leader_ yang terpilih. Dalam keadaan seperti itu, sebuah klaster tidak dapat membuat perubahan apa pun ke kondisi saat ini, yang menyebabkan tidak ada _Pod_ baru yang dapat dijadwalkan.
```

Comment:
Kubernetes API objects should be written in CamelCase, instead of italic, you can check the list here: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```suggestion
  Kinerja dan stabilitas dari klaster sensitif terhadap jaringan dan _IO disk_. Kekurangan sumber daya apa pun dapat menyebabkan _timeout_ dari _heartbeat_, yang menyebabkan ketidakstabilan klaster. Etcd yang tidak stabil mengindikasikan bahwa tidak ada _leader_ yang terpilih. Dalam keadaan seperti itu, sebuah klaster tidak dapat membuat perubahan apa pun ke kondisi saat ini, yang menyebabkan tidak ada Pod baru yang dapat dijadwalkan.
```

Reply:
Oh I see, thank you. I will change it.